api/src/main/scala/com/gu/auth/AWSCredentials.scala

```diff
@@ -1,11 +1,15 @@
 package com.gu.auth
 
 import com.amazonaws.auth.profile.ProfileCredentialsProvider
-import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain}
+import com.amazonaws.auth.{ AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain }
+import software.amazon.awssdk.auth.credentials.{ AwsCredentialsProviderChain => V2AwsCredentialsProviderChain, ProfileCredentialsProvider => V2ProfileCredentialsProvider, DefaultCredentialsProvider => V2DefaultCredentialsProvider }
 
 object AWSCredentials {
   val awsCredentials = new AWSCredentialsProviderChain(
     new ProfileCredentialsProvider("discussion"),
-    new DefaultAWSCredentialsProviderChain()
-  )
+    new DefaultAWSCredentialsProviderChain())
+  val awsCredentialsV2 = V2AwsCredentialsProviderChain.builder()
+    .addCredentialsProvider(V2ProfileCredentialsProvider.create("discussion"))
+    .addCredentialsProvider(V2DefaultCredentialsProvider.create())
+    .build()
```

Comment:
the new version of akka sqs wants to use the AWS SDK V2 so I added credentials specifically for that, I didn't think it was worth upgrading everything else to V2 right now

Reply:
👍